packages/migrate/test/migrate.test.ts

```diff
@@ -147,6 +147,29 @@ describe('fix', () => {
       project.dispose();
     });
 
+    test('annotate with strict types', async () => {
+      // since we are no longer doing a file conversion input and output are the same
+      const [inputs, outputs] = prepareInputFiles(project, ['gts/with-strict-types.gts']);
+
+      const input: MigrateInput = {
+        projectRootDir: project.baseDir,
+        packageDir: project.baseDir,
+        filesToMigrate: inputs,
+        reporter,
+        mode: 'drain',
+      };
+
+      for await (const _ of migrate(input)) {
+        // no ops
+      }
+
+      expectFile(outputs[0]).contains('export const a: number | string = 0;');
```

Comment:
Is this just a backup to the snapshot?

Reply:
Yes. It's one easier to i debug as the assertion is succinct. Secondly it helps prevent in those cases a snapshot is updated with bad data.